src/components/table/cell/style.css

```diff
@@ -25,9 +25,9 @@ smoothly-table-expandable-row[open] > div :host:last-of-type smoothly-icon {
 }
 :host > div {
 	display: grid;
+	grid-template-columns: 1fr auto;
 	grid-auto-flow: column;
```

Comment:
This might be unessesary with the template columns rule.

Reply:
Without it the expandable cell/row arrow takes up half the space of the cell.